host.go

```diff
@@ -265,6 +267,11 @@ func (h *Host) Create(name string) error {
 		return err
 	}
 
+	_, err = mos.DetectRuntime(h.Driver)
```

Comment:
I think this should come after `SaveConfig` (actually after waiting for SSH to be up, but that code isn't merged in yet) as it is technically the first step in attempting provisioning.